Remove license - (3 bis) : moving limits to an existing constants file

guilpier-code opened the pull request:
Before reviewing this PR, please until PR #218 is merged into branch feature/remove_license

When reviewing, please have a particular care on how the content of limits.h/cpp was moved to constant.h.
This element is to be discussed.

guilpier-code committed:
Remove license : removing proxy

guilpier-code committed:
Remove proxy : removing cencode

guilpier-code committed:
Remove proxy : removing decode

guilpier-code committed:
Remove proxy : removing decrypt

guilpier-code committed:
Remove proxy : finally removing hello_web

guilpier-code committed:
Remove proxy : finally removing host-properties

guilpier-code committed:
Remove proxy : finally removing "license.h"

guilpier-code committed:
Remove proxy : finally removing keys

guilpier-code committed:
Remove license : moving limits

guilpier-code committed:
Remove license activation keys

guilpier-code committed:
Remove proxy : removing useless window enabling

guilpier-code commented on src/libs/antares/constants.h:
```diff
@@ -44,6 +44,11 @@
 /*! */
 /* # define ANTARES_USE_GLOBAL_MAXIMUM_COST 0 */
 
+// The maximum number of areas allowed in a study
```
@flomnes says :
It is my opinion that there should be no arbitrary limit on the number of areas or clusters per area.

While I agree that human-made studies will almost never go beyond these limits, computer-generated studies could.

guilpier-code commented on src/analyzer/atsp/load.cpp:
```diff
@@ -205,7 +205,7 @@ bool ATSP::loadFromINIFile(const String& filename)
 
                 pArea.push_back(info);
-                if (License::Limits::areaCount and pArea.size() > License::Limits::areaCount)
+                if (MAX_NUMBER_OF_AREAS and pArea.size() > MAX_NUMBER_OF_AREAS)
```
@flomnes says : 
remove this condition

flomnes reviewed (changes requested):
Remove conditions based on max number of clusters and areas. Remove `MAX_NUMBER_OF_AREAS` and `MAX_NUMBER_OF_THERMAL_CLUSTERS_PER_AREA`.

guilpier-code committed:
Remove license : moving limits (part 2)

MAX_NUMBER_OF_AREAS and MAX_NUMBER_OF_THERMAL_CLUSTERS_PER_AREA are completely removed from the code

guilpier-code committed:
Remove license : moving limits (part 2) - correction

Forgot to remove the #include "constant.h" in the previous commit.
Now done.
